include/lmms_basics.h

```diff
@@ -33,6 +33,9 @@
 #include <cstdint>
 #include <array>
 
+#include <cmath>
```

Comment:
Why are you adding these headers here,? I don't see any purpose for these.

Reply:
It spreads into any other file that includes this, we automatically get cmath and algorithm. It's not bad, but it allows us to assume we can call basic math functions and anything in algorithm for generally any file.

Reply:
Ohh i see.

Reply:
Good catch. They seem to have been leftovers from previous code changes before `SampleFrame` was moved into its own file. Removed with commit c8f3a007a77.